README.md

```diff
@@ -17,17 +18,59 @@ It supports several languages:
 * Rust
 * Typescript
 
-## How to build it
+## Supported Metrics
 
-```console
-# Fetch the tree-sitter dependencies
-git submodule init
-git submodule update --recursive
+- CC
+- SLOC
+- LLOC
+- HALSTEAD
+- NEXITS
+- NARGS
```

Comment:
Could you add a short description of each?

Reply:
Done.